### ISSUE TITLE
[BUG]: Script execution is prevented by line-comments

### ISSUE THREAD

**Issue by tho-gru-38** [usebruno/bruno#3216]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

Hi,

I am using the portable version(s) of Bruno. Usually I am on 1.28.0 but I can reproduce it in 1.30.1 too. I run Bruno on Windows 10.

Today I have some issue with line comments, the comments starting with `//`. The `/* comment */` does not cause trouble.

In the attached request, you can see that each section looks similar to:

```
// /* comment */
console.log("Pre Request - test message");
```

When executing this request no messages appear in the console. If I delete the `//` everything works fine. The other option to prevent this error is to convert the EOL-Character from Unix (LF) to Windows (CRLF).

If you have issues to reproduce the issue try to add some characters to the messages and save the request inside Bruno. After that the bru file contains the Unix-EOL (LF) again. I this state the scripts are not executed and no messages appear at the console.

Finally: When changing the scripts to

```
/* comment */
console.log("Pre Request - test message");
```

the messages appear on the console independent of the EOL character.

With longer scripts and line comments in the middle of the script I detected that the script stops executing at the first line comment.

Kind Regards
Thomas

### .bru file to reproduce the bug

```
meta {
  name: Script-with-Comment
  type: http
  seq: 4
}

get {
  url: https://jsonplaceholder.org/users
  body: none
  auth: none
}

script:pre-request {
  /* comment */
  console.log("Pre Request - test message");
}

script:post-response {
  /* comment */
  console.log("Post Response - test message");
}

tests {
  /* comment */
  console.log("Tests - test message");
}
```

### Screenshots/Live demo link

Not working:
![image](https://github.com/user-attachments/assets/62690574-63d8-446b-a14d-c1faf42d997c)

Working (no line comments `//`):
![image](https://github.com/user-attachments/assets/0528bd4a-86ee-45a8-a595-89c0e99df400)

**Comment by anusree-bruno** [comment]
I was able to reproduce this. Our team will be working on fixing this.

**Comment by helloanoop** [comment]
Merged PR #3462 
This is set to be released in `v1.35.0` tomorrow

**Comment by tho-gru-38** [comment]
Thanks for the fix. :-)
